egs/tedlium3/asr1/RESULTS.md

```diff
@@ -0,0 +1,50 @@
+# VGGBLSTMP(elayers=4, eunits=1024) + joint ctc decoding + lm rescoring
+- Environments (obtained by `$ get_sys_info.sh`)
+  - date: `Tue Jun 11 11:22:28 JST 2019`
+  - system information: `Linux chikaku1.sp.m.is.nagoya-u.ac.jp 3.10.0-862.14.4.el7.x86_64 #1 SMP Wed Sep 26 15:12:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`
+  - python version: `Python 3.6.5`
+  - espnet version: `espnet 0.3.1`
+  - chainer version: `chainer 5.0.0`
+  - pytorch version: `pytorch 1.0.0`
+  - Git hash: `2679f69a46f5cfa103325ceffb53793d6af4af2b`
+
+- Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link: (put the model link manually. please contact Shinji Watanabe <shinjiw@ieee.org> if you want a web storage to put your files)
```

Comment:
Could you upload your model to the Google drive?
I sent you an invitation.

Reply:
Sure. Thank you.